front-end/src/_redirects

```diff
@@ -0,0 +1 @@
+/*    /index.html    200
```

Comment:
I think this was to get netlify to redirect everything through the index so the angular router worked properly.